conductr_cli/resolvers/resolvers_util.py

```diff
@@ -9,7 +8,6 @@ def is_local_file(uri, require_bundle_conf):
     return parsed.scheme == 'file' and os.path.exists(parsed.path) and (
         os.path.isfile(parsed.path) or (
             os.path.isdir(parsed.path) and
-            detect_format_dir(parsed.path) is not None and
```

Comment:
I took another look and I think you are doing the right thing here, so just a manual test as discussed. I believe it will work. This issue has the exact manual test you want: https://github.com/typesafehub/conductr-cli/pull/433